README.md

```diff
@@ -55,10 +55,11 @@ func main() {
 
 ## Running the Demo
 
-To run the demo, pull down the source code, install Go, and run `make test-verify`.
-This will use openssl to generate a certificate chain.
+To run the demo, pull down the source code, install Go, and run `make test-verify`. This will use openssl to generate a
```

Comment:
IIRC we try to stay within 80 characters per line for our projects, Go included. I think your editor may be configured for 120, and automatically changed this?

Reply:
yes exactly, sorry about that 🤦